etc/profile-m-z/tutanota-desktop.profile

```diff
@@ -9,23 +9,40 @@ include globals.local
 noblacklist ${HOME}/.config/tuta_integration
 noblacklist ${HOME}/.config/tutanota-desktop
 
+ignore dbus-user none
```

Comment:
Suggestion: move this line and place it right above dbus-user filter

Reply:
> Suggestion: move this line and place it right above dbus-user filter

Shouldn't all ignores go below `include globals.local`?

Reply:
> Shouldn't all ignores go below include globals.local?

Well, that's actually not very clear if one goes with the guidance of the profile.template. We do have a fair share of profiles that indeed place ignores higher up. And then there's others (like the firefox redirects for example) who place it below added dbus-user filter rules. Personally I like to keep the 'section' logic intact and place them accordingly. As long as the desired end goal is attained I don't hold strong opinions on this. To clarify:

foo.profile
```
[...]
include foo.local
[...]
protocol unix,inet,inet6,netlink
[...]
```

foo.local
```
ignore protocol
protocol unix,inet
```

versus

foo.local
```
protocol unix,inet
ignore protocol
```

have a different impact on the list of allowed protocols in a sandbox.

Perhaps we can leave it here as is, un-confuse @pirate486743186 about this, and go over all profiles in a separate PR if streamlining ignores is desired? Anyway, I don't see my suggestions as something that would keep back merging this, which is why I didn't opt to 'request changes' for these nits. But I should have communicated that more explicitly.

Reply:
@glitsj16 [on Feb 26](https://github.com/netblue30/firejail/pull/5325/files#r1118021912):

> > Shouldn't all ignores go below include globals.local?
> 
> Well, that's actually not very clear if one goes with the guidance of the
> profile.template. We do have a fair share of profiles that indeed place
> ignores higher up. And then there's others (like the firefox redirects for
> example) who place it below added dbus-user filter rules. Personally I like
> to keep the 'section' logic intact and place them accordingly. As long as the
> desired end goal is attained I don't hold strong opinions on this.

I see, thanks for the details.

> Perhaps we can leave it here as is, un-confuse @pirate486743186 about this,
> and go over all profiles in a separate PR if streamlining ignores is desired?
> Anyway, I don't see my suggestions as something that would keep back merging
> this, which is why I didn't opt to 'request changes' for these nits. But I
> should have communicated that more explicitly.

Sure, let's leave it as is and maybe discuss it after this PR.